python_cli/sniffle/sniffle_hw.py

```diff
@@ -112,10 +115,11 @@ def __init__(self, serport=None, logger=None, timeout=None):
                     serport = find_catsniffer_v3_serport()
                     if serport is None:
                         raise IOError("Sniffle device not found")
-                else:
+                elif baudrate is None:
                     baud = 921600
         elif is_cp2102(serport):
-            baud = 921600
+            if baudrate is None:
+                baud = 921600
```

Comment:
I think this could be removed as part of a reversion to 2Mbaud everywhere as requested in #101 , and then just letting people use the new baudrate option this introduces if they want 921600.

Reply:
Without that, it isn't working with the current code, as other hw doesn't support 2Mbaud.